diff -Nur pnm2ppa-1.10pre1.orig/Makefile pnm2ppa-1.10pre1/Makefile
--- pnm2ppa-1.10pre1.orig/Makefile	Mon Feb 11 16:49:11 2002
+++ pnm2ppa-1.10pre1/Makefile	Tue Jun 11 23:19:56 2002
@@ -5,9 +5,9 @@
 # paths to locations where the pnm2ppa binary, the configuration file,
 # and the man page should be installed.   On some systems (like Solaris)
 # the install command requires the syntax BINDIR="-f /usr/local/bin".
-BINDIR="/usr/local/bin"
-CONFDIR="/etc"
-MANDIR="/usr/local/man/man1"
+#BINDIR="/usr/local/bin"
+#CONFDIR="/etc"
+#MANDIR="/usr/local/man/man1"
 
 #compiler and linker options:
 CC=gcc
