diff -Nur pnm2ppa-1.10pre1.orig/pnm2ppa.conf pnm2ppa-1.10pre1/pnm2ppa.conf
--- pnm2ppa-1.10pre1.orig/pnm2ppa.conf	Tue Jun 11 23:14:48 2002
+++ pnm2ppa-1.10pre1/pnm2ppa.conf	Tue Jun 11 23:16:00 2002
@@ -12,7 +12,6 @@
 # will be used.   The printer version can also be set with the command line
 # option e.g., "-v 720".
 
-version 0       # REMOVE THIS ENTRY to make 720 the default printer model!!!
 #version  720	# 710, 712, 722 also acceptable
 #version  820
 #version 1000
@@ -30,7 +29,7 @@
 # configuration  file (/etc/pnm2ppa.conf), and not from configuration files 
 # specified  with the pnm2ppa option "-f".
 
-#silent 1
+silent 1
 #verbose 1
 
 #---------set the margins of the printed page-------------------
